01-ruby-fundamentals/06-oo-methods.md

```diff
@@ -96,3 +96,26 @@ puts result
 ```
 
 **Methods can often be referred to as _messages_. When we send a message, we expect a reply.**
+
+## Method Vocabulary
+
+This vocab list will reference the following code:
+
+```ruby
+def sum(a, b, c)
+  result = a + b + c
+  return result
+end
+
+number = sum(5, 17, 106)
+puts number
+```
+
+Term             | Definition | Example&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
I feel like this is a description rather than a definition. Though that fact does not impact the comprehension of the table so I don't think it needs to be changed.